targets/lenovo-x1-carbon.nix

```diff
@@ -173,6 +173,8 @@
 
             ghaf = {
               host.kernel_hardening.enable = false;
+              
```

Comment:
Continuing on mutually exclusive - both can be `false` both both can't be `true` at the same time. Either can be `true` separately.